index.json

```diff
@@ -679,7 +725,8 @@
       "currentSpecification": "trusted-types"
     },
     "nightly": {
-      "url": "https://w3c.github.io/webappsec-trusted-types/dist/spec/"
+      "url": "https://w3c.github.io/webappsec-trusted-types/dist/spec/",
+      "repository": "https://github.com/w3c/webappsec-trusted-types/"
```

Comment:
Spurious trailing slash (none of the other repo urls have it)

Reply:
The inconsistency is in Specref. The code is enough to deal with all the nightly URLs that we have in the list for now, so we could perhaps just ignore repository info from Specref. Or we keep on using Specref and fix the info there, which would potentially create further hiccups in the future but would also improve alignment between projects.

Reply:
if we don't need specref, I think we're better off limiting the dependency; fixing specref for that one wouldn't hurt though :)